content/docs/use-cases/managing-datasets.md

```diff
@@ -0,0 +1,38 @@
+# Managing Datasets Effectively
+
+**Problem**: managing datasets can be a mess. Are file names important? Where do
+you store them? How do you keep track of data [revisions][versioning]? How can
+they be shared and who can access them? How should multiple projects reuse them?
+Can they be published or plugged into production systems?
+
+> [Data lifecycle](https://www.usgs.gov/products/data-and-tools/data-management/data-lifecycle):
+> Plan > Acquire > Process > Analyze > Preserve > Publish/Share
+
+**Solution**: no ad-hoc file names, handle large files and directories (in Git),
```

Comment:
Here's that first draft we talked about @shcheklein.

Reply:
Thouoghts?

Reply:
We went into data versioning and dataset registries?

This should be about efficient, performant data storage.

Let's coin a term here (like we did with Datasets Registry)? 

E.g.

Data Cache Layer (I would not worry about people mixing Cache and dvc cache - they are not familiar with that DVC implementation detail)
Datasets Cache
Datasets Cache Layer

I would think in this direction ^^

This what is it about. We could use a diagram where we show remote, cache (mention that is it on NAS, Ceph, EFS, etc), clients attached vs regular setup where cache is per client.

Reply:
> This should be about efficient, performant data storage.

OK how about title **Effective Dataset Storage**?

*Dataset Cache Layer* sounds nice too but seems indirect/ hard to relate to. In fact I'm not sure I get how it is some kind of layer. Between what? (Storage systems and your project I guess?)

Reply:
(Updated title and intro for now)

Reply:
Between remote storage and your projects. Yes. It is _cache_ by definition (that's why it's called this way in DVC). But when we make it global we kinda make it an explicit part of your infrastructure and it becomes Objects Cache, Cache Layer, etc. Similar to the regular Datasets Storage that can mean S3 remote (and that's why Datasets Storage is nice, but I'm hesitant a bit since it's not enough to differentiate it from remotes, and it's not enough to emphasize the goal - faster access - that's what any cache is about - more expensive storage, smaller, faster).

Reply:
OK I think I got this part now.